sdk/translation/Azure.AI.Translation.Document/samples/CodeSamples/SampleListAllSubmittedJobs.cs

```diff
@@ -0,0 +1,57 @@
+﻿// <copyright file="SampleListAllSubmittedJobs.cs" company="Microsoft Corporation">
+// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+
+using Azure;
+using Azure.AI.Translation.Document;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+/**
+FILE: SampleListAllSubmittedJobs.cs
+DESCRIPTION:
+    This sample demonstrates how to list all the submitted translation jobs for the resource and
+    wait until done on any jobs that are still running.
+USAGE:
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_DOCUMENT_TRANSLATION_ENDPOINT - the endpoint to your Document Translation resource.
+    2) AZURE_DOCUMENT_TRANSLATION_KEY - your Document Translation API key.
+**/
+
+namespace DocumentTranslatorSamples
+{
+    class SampleListAllSubmittedJobs
+    {
+        public async static Task RunSampleAsync()
```

Comment:
very similar to: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_OperationsHistoryAsync.cs

Although in the one we already have, we are doing extra calculations, just to also show what people can do when looking at history.

Let me know if you will prefer to still have this one

Reply:
Removed (Duplicate).